velox/docs/functions/spark/math.rst

```diff
@@ -101,6 +101,19 @@ Mathematical Functions
     Returns ``x`` rounded down to the nearest integer.
     Supported types are: BIGINT and DOUBLE.
 
+.. spark:function:: hex(x) -> varchar
+
+    Converts ``x`` to hexadecimal.
+    Supported types are: BIGINT, VARBINARY and VARCHAR.
+    If the argument is a VARBINARY or VARBINARY, the result is string where each input bytes is represented using 2 hex characters.
+    If the argument is a positive BIGINT, the result is a hex representation of the number,
+    if the argument is a negative BIGINT, the result is a hex representation of the number which treated as two's complement (up to 16 characters). ::
```

Comment:
Thank you for updating the documentation. 

typo: "which treated"

> (up to 16 characters)

This applies to positive BIGINT, right? Negative BIGINT always returns 16 characters, no?

Could you add an example for -17 as well?

Reply:
> This applies to positive BIGINT, right? Negative BIGINT always returns 16 characters, no?

You are right.